sql/sql_tablespace.cc

```diff
@@ -510,6 +515,8 @@ bool Sql_cmd_create_tablespace::execute(THD *thd) {
   // - Disallow encryption='y', if SE does not support it.
   if (hton->flags & HTON_SUPPORTS_TABLE_ENCRYPTION) {
     tablespace->options().set("encryption", encrypt_type);
+    tablespace->options().set("explicit_encryption",
+                              m_options->encryption.str ? true : false);
```

Comment:
Only create path is handled. ALTER TABLESPACE should also set this? :thinking:

Reply:
I have added handling the ALTER path.